packages/lists/i18n/en.i18n.json

```diff
@@ -0,0 +1,9 @@
+{
+  "Lists": {
+    "methods": {
+      "remove": {
+        "lastPublicList": "Cannot delete the last public list."
```

Comment:
This looks great to me - I love the namespacing. Although, this is one place there could be a lot of benefit to having all of the translations in one file - that way you could just give the file to a translation service and get all of the different languages, right?